src/redux/FormComponents/Form.tsx

```diff
@@ -16,8 +16,21 @@ import { FormItem } from './FormItem';
 
 const REQUIRED_FIELD = 'REQUIRED_FIELD';
 const styles = () => ( {
-    container: {
-        width: '100%',
+    container: {        
```

Comment:
Form component should be generic. Place paddings/margins in crypto-frontend.

Reply:
Removed specific styles from generic form container and made into separate classes